### DISCUSSION TITLE
How can I get the iconset/themes to work from AngelV's PR?

### DISCUSSION THREAD

**Discussion by cspencer49519** [Jyers/Marlin#585] [Q&A] [answered]
I've replaced the ico file in DWIN_SET with 9.new+denoised.ICO, renamed the file to 9.ICO, but I still have the same icon set? Also, I'm not sure how to use any of these themes? I've compiled the latest and I do have color options, but nothing related to the theme?

PR is here: https://github.com/Jyers/Marlin/pull/528

**Comment by gothcha** [comment, marked as answer]
There is a link on thread #479

**Reply by cspencer49519** [reply]
Thank you so much! I knew I seen it somewhere but I couldn't find it again to save my life!
